packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -430,6 +430,75 @@ abstract class ScrollView extends StatelessWidget {
 /// ```
 /// {@end-tool}
 ///
+/// {@tool dartpad --template=stateful_widget_material}
+///
+/// [ListView] is often used to display items in a scrolling widget. If items
+/// are added to the top of a [ListView], the scroll position moves, as it is
+/// meant to only grow in the direction of the scroll axis.
+///
+/// To have a scrolling widget that grows in both directions along its scroll
+/// axis, use a [CustomScrollView]. This sample code shows a scroll view that
+/// contains two [SliverList]s, and by setting the [CustomScrollView.center]
+/// to the key of the second list, it can grow in both directions along its
+/// scroll axis without moving the scroll position.
+///
+/// ```dart
+/// List<int> top = [];
+/// List<int> bottom = [0];
+///
+/// @override
+/// Widget build(BuildContext context) {
+///   const Key centerKey = ValueKey('second-sliver-list');
```

Comment:
It would be helpful if we consistently referred to the second sliver list as the "bottom".

Reply:
Done.